src/util/structured_data.cpp

```diff
@@ -0,0 +1,225 @@
+// Copyright 2016-2020 Diffblue Limited. All Rights Reserved.
+
+#include "structured_data.h"
+#include "range.h"
+#include "string_utils.h"
+#include <algorithm>
+
+std::string capitalize(const std::string &str)
+{
+  std::string capitalized = str;
+  capitalized[0] = toupper(capitalized[0]);
```

Comment:
This errors on the empty string

Reply:
This is a little fun btw - accessing the 0th element is not undefined behaviour, but modifying it is!